docs/map.json

```
@@ -539,7 +539,7 @@
               "id": "packages-devkit",
               "tags": ["create-your-own-plugin"],
               "file": "",
-              "path": "/packages/devkit/index",
+              "path": "/packages/devkit",
```

Could you change this to:
```suggestion
            "path": "/packages/devkit/documents/index",
```

So we can keep the same intent?